src/carp/campaign/campaign.entity.ts

```diff
@@ -12,4 +13,11 @@ export default class Campaign {
   @ManyToMany(() => Character)
   @JoinTable()
   characters: Character[];
+
+  @ManyToMany(() => Monster)
+  @JoinTable()
+  monsters: Monster[];
+
+  @Column()
+  dungeon: string;
```

Comment:
Just to highlight that the chat about serializing and deserializing could be adopted here too.